src/settings.cpp

```diff
@@ -44,6 +44,10 @@ std::wstring Settings::ms_appVersion;
 std::wstring Settings::ms_appBuildVersion;
 std::string Settings::ms_DSAPubKey;
 
+void *Settings::ms_customConfigData = nullptr;
+win_sparkle_config_read_t Settings::ms_currentConfigRead = Settings::DefaultConfigRead;
```

Comment:
Missing `&` in front of the assigned value (`… = &Settings:…`) unless I'm mistaken.